docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -43,13 +43,26 @@ Does this look interesting? Read on!
 
 NOTE: the `list()` method might be surprising at first. It takes fragments of HQL (JP-QL) queries and contextualizes the rest. That makes for very concise but yet readable code.
 
+NOTE: what you just see is the link:https://www.martinfowler.com/eaaCatalog/activeRecord.html[active record pattern], sometimes just called the entity pattern.
```

Comment:
```suggestion
NOTE: what was described above is a essentially the link:https://www.martinfowler.com/eaaCatalog/activeRecord.html[active record pattern], sometimes just called the entity pattern.
```

Reply:
`is a essentially the` isn't the `a` needed ?

Reply:
the `a` was indeed a mistake :)